source/extensions/filters/network/connection_limit/connection_limit.cc

```diff
@@ -80,11 +80,13 @@ Network::FilterStatus Filter::onNewConnection() {
     absl::optional<std::chrono::milliseconds> duration = config_->delay();
     if (duration.has_value() && duration.value() > std::chrono::milliseconds(0)) {
       delay_timer_ = read_callbacks_->connection().dispatcher().createTimer([this]() -> void {
-        read_callbacks_->connection().close(Network::ConnectionCloseType::NoFlush);
+        read_callbacks_->connection().close(Network::ConnectionCloseType::NoFlush,
+                                            "connection_limit_close_over_limit");
```

Comment:
Thanks for adding the details.  
How about just "over_connection_limit"?
/wait-any